packages/kbn-securitysolution-grouping/src/containers/query/index.ts

```diff
@@ -69,6 +76,13 @@ export const getGroupingQuery = ({
             size,
           },
         },
+        // this agg will return a count when the bucket is of a missing field value
+        // this is so that we have a flag in the UI for the "missing" group
+        nullGroup: {
+          missing: {
+            field: groupByFields[0],
+          },
+        },
```

Comment:
@stephmilovic I am not sure we need this additional sub-aggregation. We are asking for the same "missing" information in the `terms` or `multi_terms` aggregation up there.

Couldn't we retrieve the "empty" counts from this values agg? we could find the "empty" bucket key using `getFieldTypeMissingValue` right? Or am I missing something?

This new agg probably makes it easier for the UI to find the info in the response, but it looks like we are asking for the same thing twice, adding unnecessary load to ES only for the convenience of the UI logic. 

I think we should ask for the "missing" in the `terms` and `multi_terms` OR ask for the "missing" in an additional aggregation, but doing both things looks like overkill to me.

Do we have to merge this PR urgently?
cc @YulNaumenko